automation/src/test/scala/org/broadinstitute/dsde/workbench/hamm/automation/WorkflowCostTest.scala

```diff
@@ -1,11 +1,11 @@
-//package org.broadinstitute.workbench.hamm
+//package org.broadinstitute.dsde.workbench.hamm
```

Comment:
I added a ticket to add automation - I think we'll have to talk about how best to do it since we aren't using GRPC anymore - https://broadworkbench.atlassian.net/browse/AP-151

Reply:
I think the only difference from what it was before is how we call the APIs. So instead of calling hamm api with grpc client, we're using a http client

Reply:
This PR is already really long and I don't want to add more to it to review - I would like to merge it and get out of this prolonged review cycle

Reply:
I think the fact that this PR got so many things piled together is the reason why review is hard and it takes this long, which I mentioned in the other comment.

It's not ideal that we're deleting tests and not adding them back, but I'm fine with it being added in a followup PR at this point.